src/Orders.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
+
+const Orders = props => {
+  const [orders, setOrders] = useState(0);
+
+  function handleOneOrder() {
+    setOrders(orders + 1);
+  }
+
+  return (
+    <div>
```

Comment:
You don't need to wrap this element in a `<div>`

If you do, you end up having HTML that looks like
```
<ul>
  <div><li>....</li></div>
  <div><li>....</li></div>
  <div><li>....</li></div>
</ul>
```
when it should just be
```
<ul>
  <li>...</li>
  <li>...</li>
  <li>...</li>
</ul>
```